DQM/SiPixelPhase1Heterogeneous/plugins/SiPixelPhase1MonitorRecHitsSoA.cc

```diff
@@ -0,0 +1,204 @@
+// -*- C++ -*-
+///bookLayer
+// Package:    SiPixelPhase1MonitorRecHitsSoA
+// Class:      SiPixelPhase1MonitorRecHitsSoA
+//
+/**\class SiPixelPhase1MonitorRecHitsSoA SiPixelPhase1MonitorRecHitsSoA.cc 
+*/
+//
+// Author: Suvankar Roy Chowdhury, Alessandro Rossi
+//
+#include "DataFormats/Math/interface/approx_atan2.h"
+#include "CUDADataFormats/Common/interface/Product.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "HeterogeneousCore/CUDACore/interface/ScopedContext.h"
```

Comment:
I'd suggest to remove unnecessary headers (and packages from the `BuildFile`). At least these headers
```diff
-#include "CUDADataFormats/Common/interface/Product.h"
-#include "FWCore/ServiceRegistry/interface/Service.h"
-#include "HeterogeneousCore/CUDACore/interface/ScopedContext.h"
```
and packages
```diff
-<use name="CUDADataFormats/Common"/>
-<use name="CUDADataFormats/SiPixelCluster"/>
-<use name="HeterogeneousCore/CUDACore"/>
-<use name="DataFormats/SiPixelDigi"/>
```
appear to be unused now.